torchdynamo/variables/misc.py

```diff
@@ -63,6 +63,8 @@ def call_method(
         inner_fn = self.const_getattr(self, name)
         if inner_fn is object.__init__:
             return LambdaVariable(identity, **options)
+        elif inner_fn is object.__getattribute__:
+            unimplemented(f"Cant handle object getattribute")
```

Comment:
Not sure how to handle this.

Reply:
Check out the discussion on https://github.com/pytorch/torchdynamo/pull/454/files?

Reply:
Added a special handling for PreTrainedConfig